build.gradle

```diff
@@ -29,7 +29,9 @@ if (!repositories) {
       if ("$nettyVersion".endsWith("+")) {
         maven {
           name "Netty Snapshots"
+          // This will eventually need to become https://central.sonatype.com/repository/maven-snapshots/
           url "https://oss.sonatype.org/content/repositories/snapshots"
```

Comment:
Can it change now? We can even try it as a separate PR and check it's working as expected by triggering corresponding GitHub Action

Reply:
I looked and there doesn't appear to be netty artifacts there right now. Maybe I looked in the wrong spot.